js/company.js

```diff
@@ -0,0 +1,91 @@
+async function getCompanyHistory(symbol) {
+    try {
+        const url =
+            "https://stock-exchange-dot-full-stack-course-services.ew.r.appspot.com/api/v3/historical-price-full/" +
+            symbol + "?serietype = line";
+
+        const response = await fetch(url);
+        const result = response.json();
```

Comment:
What is result? think of the data you're storing in here, it will help you with naming variables in more meaningful way.